src/app/proposals/[id]/page.tsx

```diff
@@ -305,16 +305,22 @@ const PageWithProposal = (proposal: ParsedProposal) => {
         )}
       </div>
       <div className="flex flex-row justify-between">
-        <div className="flex flex-row gap-x-6">
+        <div className="flex flex-row gap-x-8 basis-1/2">
+          <ProposalQuorum blockNumber={proposal.blockNumber} />
           <MetricsCard
             title="Snapshot"
             amount={formatNumberWithCommas(snapshot?.toString() || '-')}
             fiatAmount="Taken at block"
+            borderless={true}
           />
-          <MetricsCard title="State" amount={proposalStateHuman} />
         </div>
         <div>
-          {proposalState === ProposalState.Active && (
+          {!isConnected && (
+            <Button variant="secondary" data-testid="VoteOnChain">
```

Comment:
When the user is not connected - this must call the ConnectWorkflow -> see DelegationSection.tsx:82
![image](https://github.com/user-attachments/assets/fa5f347a-85af-4eb6-9c92-9487a5f148cb)